lib/monitor_resource_type.rb

```diff
@@ -41,7 +41,6 @@ def self.create(klass, stuck_pulse_info, num_threads, types)
             yield r
           rescue => ex
             Clog.emit("Monitoring job has failed.") { {monitoring_job_failure: {resource: r.resource, exception: Util.exception_to_hash(ex)}} }
-            r.resource.incr_checkup if r.resource.respond_to?(:incr_checkup)
```

Comment:
With the switch to use partitioned monitor, false positives shouldn't be an issue. Are you sure this won't introduce false negatives (cases where we should `incr_checkup` but will no longer do so)? If `incr_checkup` isn't called here, what else will call it in cases where it should be called?

Reply:
I have removed this (the last) commit.